Doc/library/http.cookies.rst

```diff
@@ -151,6 +152,18 @@ Morsel Objects
    send the cookie along with cross-site requests. This helps to mitigate CSRF
    attacks. Valid values for this attribute are "Strict" and "Lax".
 
+   The attribute :attr:`partitioned` indicates to user agents that these
+   cross-site cookies *should* only be available in the same top-level context
+   that the cookie was first set in. For this to be accepted by the user agent,
+   you **must** also set both ``Secure`` and ``Path=/``.
```

Comment:
I updated the wording here to clarify that Secure is required... but the CHIPS spec doesn't explicitly say anything about Path= though Path=/ appears in all of its examples.  What wording should be used regarding Path, I'm not sure how I've phrased this is wholly accurate.